tests/base/utils.py

```diff
@@ -92,3 +95,29 @@ def test_get_m2m_mixed(user0, user1, userX):
         get_user_model().objects.filter(pk__in=[user1.pk]),
         get_user_model().objects.filter(pk__in=[userX.pk, user0.pk]),
     ) == ([user0, userX], [user1])
+
+
+def test_util_base_extension_in():
+    assert extension_in('filename.txt', ['bat', 'txt']) is True
+    assert extension_in('filename.biff', ['biff']) is True
+    assert extension_in('filename.tar.gz', ['gz']) is True
```

Comment:
What is the point of using `is True` here? Functionally, it can be removed.

Reply:
well - i would make 2 points here - this is what the existing test did under the hood, and sometimes you want to know an api returns a bool - not eg the filename or somesuch

i have no qualms either way tho - so ill update now...